qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -75,6 +75,7 @@ def _default_experiment_options(cls) -> Options:
 
         options.delays = None
         options.unit = "s"
+        options.conversion_factor = None
```

Comment:
Same comment as for T1, in my opinion this shouldn't be an experiment option

Reply:
same for above